### ISSUE TITLE
[Xamarin UWP] BalloonPopup with German characters in title/description doesn't work

### ISSUE THREAD

**Issue by SuNNjek** [CartoDB/mobile-sdk#488]
I'm trying to create a BalloonPopup that contains text in German. The German alphabet contains some "special" characters on top of all the letters in the English alphabet, i.e. ä/Ä, ö/Ö, ü/Ü and ß/ẞ. However, when either the text or the description of the BalloonPopup contains any of these characters, it doesn't get displayed on UWP.

The code I'm using to create the popup is as follows:
```csharp
_activePopup = new BalloonPopup(
	point.GetPos(),
	GetStyleForMarker(_activeMarker),
	_activeMarker.Label ?? String.Empty,
	_activeMarker.Text ?? String.Empty
);

_popupLayerDataSource.Add(_activePopup);
```
where `_popupLayerDataSource` is a `LocalVectorDataSource`

When I try to display a text without umlauts it looks like this:
![grafik](https://user-images.githubusercontent.com/20151081/153570530-4015ad63-b437-4a3d-b4f7-118b1c297f75.png)

When I change the "Hauptstrasse" into "Hauptstraße" however, nothing happens on UWP, even though the code above gets called correctly:
![grafik](https://user-images.githubusercontent.com/20151081/153570674-95341d79-f87c-49ce-bfcc-f217de3e2957.png)

On Android and iOS it works correctly:
![grafik](https://user-images.githubusercontent.com/20151081/153571055-f248ff92-5a7a-4282-aa2b-283bfba39f4b.png)

**Comment by mtehver** [comment]
@SuNNjek RC1 of SDK 4.4.5 is now available and includes the fix for this.

**Comment by mtehver** [comment]
Closing this, fix is included in 4.4.5, which is now released.
